### PR TITLE
Add Background

### PR DESCRIPTION
**Pull request by nmercadeb**

### CODE REVIEW

File: R/appStatic.R [hunk 1]
```diff
@@ -6,18 +6,43 @@
...
+#' element's name indicates the type of text it will fill. The order of these
+#' elements is important when displaying the content. Markdown sythax for text
+#' styling is supported.
 #' @param open Whether to open the shiny app project.
 #'
 #' @return The shiny app will be created in directory.
+#'
 #' @export
 #'
+#' @examples {
```
Comment (catalamarti) [inline R/appStatic.R:24]: maybe this content would fit more for a vignette than as an example?

File: R/appStatic.R [hunk 2]
```diff
@@ -111,18 +136,18 @@ copyLogos <- function(logo, directory) {
...
                      logo = NULL,
                      title = "My study",
-                     background = TRUE) {
+                     background = NULL) {
   # initial checks
   omopgenerics::assertList(choices, named = TRUE)
   omopgenerics::assertCharacter(logo, length = 1, null = TRUE)
   omopgenerics::assertCharacter(title, length = 1)
-  omopgenerics::assertLogical(background, length = 1)
+  # TODO: check background
```
Comment (catalamarti) [inline R/appStatic.R:144]: not needed

File: R/background.R [hunk 3]
```diff
@@ -15,14 +15,48 @@ createBackground <- function(background, title, logo) {
...
+            "x" = "{e$message}")
+        )
+        NULL
+      }
+    )
+  } else {
+    omopgenerics::assertCharacter(x = background, null = TRUE, named = TRUE)
+    notAllowed <- ! names(background) %in% c("header", "title", "body", "footer")
+    if (sum(notAllowed) > 0) {
+      cli::cli_warn("{background[notAllowed]} {?is/are} not allowed named for `background` and will be ignored.")
```
Comment (catalamarti) [inline R/background.R:57]: can we combine both behaviours?

File: R/background.R [hunk 4]
```diff
@@ -1,6 +1,6 @@
-
-createBackground <- function(background, title, logo) {
-  if (!background) return("")
+createBackground <- function(background = NULL,
+                             logo = NULL) {
+  if (length(background) == 0) return("")
```
Comment (catalamarti) [inline R/background.R:3]: the default background is empty?

**Review by catalamarti — CHANGES_REQUESTED** [review]

File: R/background.R (hunk 4 above)
Comment (nmercadeb) [reply R/background.R:3]: At the moment yes

File: R/background.R (hunk 3 above)
Comment (nmercadeb) [reply R/background.R:57]: What do you mean¿

File: R/appStatic.R (hunk 1 above)
Comment (nmercadeb) [reply R/appStatic.R:24]: Do I eliminate these examples¿

Comment (catalamarti) [reply R/appStatic.R:24]: Maybe you can copy that in a simple vignette (so we do not lose what you've already done) and when we have a stable behaviour for background argument we should write a proper vignette @nmercadeb

**Review by catalamarti — APPROVED** [review]

**Pull request merged**